config-files/garnet_3layer_config.yml

```diff
@@ -0,0 +1,37 @@
+KerasJson: garnet_3layer.json
+KerasH5:   garnet_3layer_weights.h5
```

Comment:
I think everything looks good now. I just have a question here. If you do end up using the config file in your test, wouldn't you use relative path here? Or is it not important?

Reply:
If you are fine leaving them like this I can merge this PR now.

Reply:
Yeah I think it's fine like this. These lines are sort of suggesting that the user should copy all relevant files into a work directory before running `hls4ml convert`.